resources/objects/accessoires/tray.tscn

```diff
@@ -21,6 +23,10 @@ radius = 0.04
 
 [node name="Tray" instance=ExtResource("1_xu6s3")]
 collision_mask = 8388615
+mass = 0.25
+freeze = true
+continuous_cd = true
```

Comment:
Is this strictly neccessray?
Same question for other accessoires.

Reply:
Yep. Frozen before first pickup - continuous cd seems to be necessary for stuff like it being moved by a patty flipper or other things